include/crm/common/results.h

```diff
@@ -107,6 +107,8 @@ enum pcmk_rc_e {
     /* When adding new values, use consecutively lower numbers, update the array
      * in lib/common/results.c, and test with crm_error.
      */
+    pcmk_rc_bad_config          = -1028,
```

Comment:
Surprised we don't have something for this already. I'm thinking we can use pcmk_rc_transform_failed for cli_config_update() failure and pcmk_rc_schema_validation for validate_xml() failure. (They both already map to CRM_EX_CONFIG.)

Reply:
Done, leaving pcmk_rc_no_input in for the moment.